UPGRADING.md

```diff
@@ -4,6 +4,9 @@
 
 ActiveAdmin v4 uses Tailwind CSS. It has **mobile web, dark mode and RTL support** with a default theme that can be customized through partials and CSS. This release assumes `cssbundling-rails` and `importmap-rails` is installed and configured in the host app. Partials can be modified to include a different asset library, e.g. shakapacker.
 
+> [!WARNING]  
+> ActiveAdmin v4 is currently incompatible with Tailwind 4. Ensure you pin your version of Tailwind to version 3 for now.
+
```

Comment:
```suggestion
ActiveAdmin v4 uses Tailwind CSS v3 (with an upgrade to v4 coming soon). It has **mobile web, dark mode and RTL support** with a default theme that can be customized through partials and CSS. This release assumes `cssbundling-rails` and `importmap-rails` is installed and configured in the host app. Partials can be modified to include a different asset library, e.g. shakapacker.
```

Reply:
I'm going to update this as I prefer to call out the version on the first line with an added note that a Tailwind v4 upgrade is coming soon. Thanks.